src/components/biggive-heading/biggive-heading.scss

```diff
@@ -7,7 +7,14 @@
 }
 
 .container {
-  @include standard-font();
+  &.biggive {
+    @include standard-font();
```

Comment:
Seemed better to make this arg-dependent rather than try to reset from a fixed font size to a variable one.